addon/components/phone-input.js

```diff
@@ -215,6 +215,10 @@ export default Component.extend({
         'countrychange',
         this._onCountryChange.bind(this)
       );
+    } catch (error) {
+      if (this.onError) {
+        this.onError(error);
+      }
```

Comment:
```suggestion
      this.onError?.(error);
```

Reply:
yeah, I was wondering about that. can we use that here? given that we were still using Promise chains and `Component.extend()` I was aiming to keep it on the safe side and use "old" syntax 😅

Reply:
Let it be safe then 😅